src/views/refund/refund.js

```diff
@@ -16,68 +16,116 @@ const styles = () => ({
   },
 });
 
-class Refund extends React.Component {
-  UNSAFE_componentWillMount() {
-    this.props.toggleRefundMode();
-  }
-
-  componentWillUnmount() {
-    this.props.toggleRefundMode();
-  }
-
-  render() {
-    const { classes, inRefundMode, goHome } = this.props;
-    return (
-      <Background>
-        <Prompt />
-        <View className={classes.wrapper}>
-          <StepsWizard
-            dark={true}
-            range={4}
-            stage={1}
-            alertOnExit={inRefundMode}
-            onExit={() => {
+const Refund = ({
+  classes,
+  goHome,
+  refundFile,
+  setRefundFile,
+  transactionHash,
+  setTransactionHash,
+  destinationAddress,
+  setDestinationAddress,
+  startRefund,
+  refundTransaction,
+  refundTransactionHash,
+  isFetching,
+}) => {
+  return (
+    <Background>
+      <Prompt />
+      <View className={classes.wrapper}>
+        <StepsWizard
+          dark={true}
+          range={3}
+          stage={1}
+          onExit={() => {
+            if (window.confirm('Are you sure you want to exit')) {
```

Comment:
I think it is necessary that we wipe the refund state when exiting, this will eliminate any chance of wired bugs where old data is still in the state.